packages/core/src/components/SelectableText.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {
+  Platform,
+  TextInput,
+  Text,
+  TextStyle,
+  I18nManager,
+  StyleSheet,
+  TextInputProps,
+  TextProps,
+  StyleProp,
+} from "react-native";
+
+type Props = {
+  style?: StyleProp<TextStyle>;
+  text?: string;
```

Comment:
Can the text be passed in as `children`? Similar to how the normal `Text` component works